index.html

```diff
@@ -1,49 +1,51 @@
+	<form autocomplete="off">
+		<input id="url" autofocus name="url" type="url" placeholder="Paste GitHub.com folder URL" required>
+    <button type="submit">&#129095; Download</button>
 	</form>
 	<footer class="text-align-center">
-		<label for="info-toggle">info</label>
-		+
+    <label for="info-toggle">info</label>
+    &#160;•&#160;
+    <label for="token-toggle">token</label>
+
 		<input type="checkbox" id="token-toggle">
-		<span>
+		<div class="token layer-top">
 			Enter
-			<a href="https://github.com/settings/tokens/new?description=Download GitHub directory&scopes=repo" target="_blank">
+			<a href="https://github.com/settings/tokens/new?description=Download GitHub Directory&default_expires_at=7&scopes=repo" target="_blank">
 				your GitHub token</a>:<br>
-			<input id="token" placeholder="Token" autocomplete="off" size="45" pattern="[\da-f]{40}|ghp_.+|gho_.+|github_pat_.+" required><br>
+			<input id="token" class="validation" type="text" placeholder="Token" autocomplete="off" size="45" pattern="[\da-f]{40}|ghp_.+|gho_.+|github_pat_.+" required><button id="clear">Clear</button><br>
 			Brought to you by the developers of <a href="https://github.com/sindresorhus/refined-github/pull/951#issuecomment-358301513" target="_blank">Refined GitHub</a>
-		</span>
+		</div>
 
-		<label for="token-toggle">token</label>
 		<input type="checkbox" id="info-toggle">
-		<div class="info">
+		<div class="info layer-top">
 			<p>GitHub doesn’t let you download a single folder from a repo, which might be necessary when you just need a few files from a large repository.</p>
 			<p>This tool will handle the download of all the files in a directory, in a single click, after you entered your token.</p>
 			<p>The download starts automatically when you visit pass the link to the GitHub directory as <code>url</code> parameter, like:</p>
-			<p><a href="/?url=https://github.com/mrdoob/three.js/tree/dev/build"><strong>download-directory.github.io</strong><code>?url=https://github.com/mrdoob/three.js/tree/dev/build</code></a></p>
+			<p><a href="./?url=https://github.com/mrdoob/three.js/tree/dev/build"><strong>download-directory.github.io/</strong><code>?url=https://github.com/mrdoob/three.js/tree/dev/build</code></a></p>
 			<p>You can also specify download filename by adding <code>filename</code> parameter, like:</p>
-			<p><a href="/?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build"><strong>download-directory.github.io</strong><code>?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build</code></a> to save the file as <strong>three-js-build.zip</strong>.</p>
+			<p><a href="./?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build"><strong>download-directory.github.io/</strong><code>?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build</code></a> to save the file as <strong>three-js-build.zip</strong>.</p>
```

Comment:
I assume that this is to make it work from a sub-directory, but that does not happen here or during development, so also unnecessary